fees/vicuna-finance/index.ts

```diff
@@ -17,21 +19,22 @@ const adapter: Adapter = {
                 const targets = ['0xad1bB693975C16eC2cEEF65edD540BC735F8608B'];
     
                 const dailyRevenue = await addTokensReceived({ options, targets, tokens });
-                const dailyFees = dailyRevenue.clone(totalFee / vaultFee);  // Total fees = protocol revenue * (7/6.95)
-                const dailyProtocolRevenue = dailyRevenue 
+                const dailyFees = dailyRevenue.clone(totalFee / vaultFee); // Total fees = protocol revenue * (7/6.95)
+                const dailyProtocolRevenue = dailyRevenue.clone(vaultFee / totalFee); // Vault share of the revenue
+                const governanceRevenue = dailyRevenue.clone(governanceShare / totalFee); // Governance share of the revenue
     
-                return { dailyFees, dailyRevenue, dailyProtocolRevenue }
+                return { dailyFees, dailyRevenue, dailyProtocolRevenue, governanceRevenue };
```

Comment:
@dailyProtocolRevenue we don't support return governanceRevenue we accept only dailyProtocolRevenue